src/applications/applications.service.ts

```
@@ -218,4 +218,83 @@ export class ApplicationsService {
+          if (field === 'serialNumber') {
+            row.push((i + 1).toString());
+          } else {
+            row.push('');
+          }
+        }
+        
+        // Application data fields
+        for (const field of appDataFields) {
+          row.push(app.applicationData && app.applicationData[field] !== undefined 
+            ? app.applicationData[field] 
+            : '');
```

ESLint

[error] Replace `app.applicationData·&&·app.applicationData[field]·!==·undefined·⏎············?·app.applicationData[field]·⏎············:·'');` with `⏎············app.applicationData·&&·app.applicationData[field]·!==·undefined⏎··············?·app.applicationData[field]⏎··············:·'',`

(prettier/prettier)